src/main/java/com/alkemy/ong/model/request/ActivityRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.alkemy.ong.model.request;
+
+import javax.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ActivityRequest {
+
+  @NotBlank(message = "Name can not be black")
+  private String name;
+
+  @NotBlank(message = "Content can not be black")
```

Comment:
MINOR: lo cambiaria a Name cannot be empty or null. Solamente porque no recuerdo haber escrito blank en ningún mensaje de error en todos estos años de desarrollo.